 Fix wrong libdir determination when cross-compiling on 64-bit host
 
diff -urbB busybox-1.1.3/applets/install.sh busybox/applets/install.sh
--- busybox-1.1.3/applets/install.sh	2006-03-23 00:16:25.000000000 +0300
+++ busybox/applets/install.sh	2008-12-25 21:56:50.000000000 +0300
@@ -19,7 +19,10 @@
 if [ "$DO_INSTALL_LIBS" != "n" ]; then
 	# get the target dir for the libs
 	# This is an incomplete/incorrect list for now
-	case $(uname -m) in
+	if [ -z $TARGET_ARCH ]; then
+	    TARGET_ARCH=$(uname -m)
+	fi
+	case $TARGET_ARCH in
 	x86_64|ppc64*|sparc64*|ia64*|hppa*64*|s390x*) libdir=/lib64 ;;
 	*) libdir=/lib ;;
 	esac
diff -urbB busybox-1.1.3/Rules.mak busybox/Rules.mak
--- busybox-1.1.3/Rules.mak	2006-05-17 22:51:06.000000000 +0400
+++ busybox/Rules.mak	2008-12-25 21:54:45.000000000 +0300
@@ -99,6 +99,7 @@
 		-e 's/cris.*/cris/' \
 		)
 endif
+export TARGET_ARCH
 
 # A nifty macro to make testing gcc features easier
 check_gcc=$(shell \
